src/JLD.jl

```diff
@@ -92,6 +95,10 @@ iscompressed(f::JldFile) = f.compress
 iscompressed(g::JldGroup) = g.file.compress
 iscompressed(d::JldGroup) = d.file.compress
 
+ismmapped(f::JldFile) = f.mmaparrays
+ismmapped(g::JldGroup) = g.file.mmaparrays
+ismmapped(d::JldGroup) = d.file.mmaparrays
```

Comment:
This is probably my fault, but this and `iscompressed` above have two methods for `JldGroup` when only one is needed.